### ISSUE TITLE
Rails 5: undefined method `logger=' for ActiveSupport::Dependencies:Module

### ISSUE THREAD

**Issue by machty** [TwP/logging-rails#22]
```
10:44:04 web.1    | ! Unable to load application: NoMethodError: undefined method `logger=' for ActiveSupport::Dependencies:Module
10:44:04 web.1    | Did you mean?  logger_name
10:44:04 web.1    | /Users/machty/.rvm/gems/ruby-2.3.0-preview1/gems/logging-rails-0.5.0/lib/logging/rails/railtie.rb:39:in `block in <class:Railtie>': undefined method `logger=' for ActiveSupport::Dependencies:Module (NoMethodError)
```

This initializer seems to be the culprit.

If `logger` has been removed from ActiveSupport::Dependencies, what should this initializer attach to in Rails 5? (For posterity, what did setting the logger on AS::Dependencies do in the first place?)

**Comment by wbotelhos** [comment]
Hi @machty, any fix?

**Comment by machty** [comment]
@wbotelhos i put this in an initializer

```
module ActiveSupport
  module Dependencies
    def self.logger=(_)
      puts "stubbed ActiveSupport::Dependencies#logger="
    end
  end
end

```

**Comment by wbotelhos** [comment]
Thank you, @machty !

It works putting in the beginning of the `config/logging.rb` file, to avoid the config loads first and runs the same error.

**Comment by SuperTux88** [comment]
I opened a pull-request to fix this without that workaround: #23
